docker/requirements-odc.txt

```diff
@@ -1,14 +1,14 @@
 # PyPi
-datacube[performance,s3]==1.8.5
+datacube[performance,s3]==1.8.6
 
 # currently only available from dea packages
 --extra-index-url="https://packages.dea.ga.gov.au"
-odc-algo==0.2.1.dev2856
-odc-cloud[ASYNC]==0.2.1.dev2856
-odc-dscache==0.2.1.dev2856
-odc-io==0.2.1.dev2856
-odc-stac==0.2.1.dev2856
-odc-stats==0.2.1.dev2856
-odc-ui==0.2.1.dev2856
+odc-algo==0.2.1.dev2954
+odc-cloud[ASYNC]==0.2.1.dev2954
+odc-dscache==0.2.1.dev2954
+odc-io==0.2.1.dev2954
+odc-stac==0.2.1.dev2954
+odc-stats==0.2.1.dev2954
+odc-ui==0.2.1.dev2954
```

Comment:
only put version constraints in requirements files when there is a reason for it, for example when trying to work around some version conflicts, or when needing at least that version of library. Having these libs pinned here as well as in constraints.txt only makes maintenance harder. Basically if you don't have a reason to pick specific version, then don't, if you do have reason add a comment so that we can keep track of why specific version was chosen and whether that reason is still valid.